server/master/src/main/java/org/apache/accumulo/master/EventCoordinator.java

```diff
@@ -53,12 +73,11 @@ public Listener getListener() {
     long lastEvent;
 
     Listener() {
-      lastEvent = eventCounter;
+      lastEvent = eventCounter.sum();
```

Comment:
This was a bug in the existing code, the fact that counter was not volatile and was read outside of a lock.